README.md

```diff
@@ -0,0 +1,146 @@
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Stock-Market Project <a name="about-project"></a>
+
+**Stock-Market Project** Stock-Market is an application that displays a list of businesses and the overview of their finances, allows the user to filter and search for details of specific business such as  (dividends, marketcap and prices).
+
+## 🛠 Built With <a name="built-with">Create React App</a>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Project setup with Create React App**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [video link](https://www.loom.com/share/Zuwaira-Sadiq-or-Loom-13-September-2023-9e51f50a5f6241d38641a9076d8eb0de?sid=ef1e4fa5-0db4-4446-a83d-fbae5a99527e)
+
+
+- [Live Demo Link](https://enchanting-fairy-a3b864.netlify.app/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Try to have these in your local machine
+
+- Git and GitHub
+- VS Code or any code editor
+
+### Prerequisites
+
+In order to run this project you need 
+- A web browser
+- Code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone https://github.com/Zuwairanajma/react-metric-app-capstone.git
+
+### Install
+
+Install this project using the cloned repo or download and use the files to your local machine.
+
+### Usage
+
+- Clone the repository first
+- Commnads to Run: 
+    - npm install
+    - npm run build
+    - npm start
+
+### Run tests
+
+- tests are included using Jest framework
+- commands to run:   npm run test 
+
+### Deployment
+```
+You can deploy this project through Github Pages
+
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Zuwaira Sadiq**
+- GitHub: [@Zuwairanajma](https://github.com/Zuwairanajma)
+- Twitter: [@SadiqJuwairiyya](https://twitter.com/SadiqJuwairiyya)
+- LinkedIn: [Zuwaira Sadiq](https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features">Animation</a>
+
+- [ ] **Changes will be made as comming project requirement.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a star to this repositiory.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"> Nelson Sakwa on Behance</a>
+
+- I would also like to thank Microverse for the guidance they made all the way up to this point. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Please give appropriate credit to the designer in this section. According to the requirements:

![image](https://github.com/Zuwairanajma/react-metric-app-capstone/assets/77069011/9489bd96-2e8e-46e2-ab47-ca47a34cf1ac)

Consider including it 🤓 🔧